Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs

```diff
@@ -2,13 +2,19 @@
 // Licensed under the MIT License. See LICENSE in the project root for
 // license information. 
 
+using Microsoft.Extensions.PlatformAbstractions;
+using System;
+
 namespace Artesian.SDK.Service
 {
     /// <summary>
     /// Artesian Constants
     /// </summary>
     public abstract class ArtesianConstants
     {
+        internal const string SDKVersion = "v2.2.1";
```

Comment:
this 'version' should be linked to the packageVersion of nuget, not a separate string. Should be somewhere in the Assembly